LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
btw I like creating the repo from GitHub and then cloning it vs. creating it locally and then pushing it, since then GitHub will create the `LICENSE` and `.gitignore` files for you, and you can immediately switch to the "initial code" branch